ruby/lib/rspec/queue.rb

```diff
@@ -398,7 +398,7 @@ def run_specs(example_groups)
           end
         end
 
-        return 0 if @world.non_example_failure
+        return 1 if @world.non_example_failure
```

Comment:
```suggestion
        return @configuration.error_exit_code || @configuration.failure_exit_code if @world.non_example_failure
```

.. would mirror what [rspec itself does](https://github.com/rspec/rspec-core/blob/ea8554afd1a2b63677c6593059fa8f2476181deb/lib/rspec/core/runner.rb#L191), not sure if we want that though.